### PR TITLE
MH forms fine-tuning

### PR DESCRIPTION
**Pull request by jnsereko**
### Not fixed (depends on OpenMRS community)

1. Follow-up session: the 'Appointments' workspace should be hidden if 'No'
remains visible even when hide is hard coded to true hence needs community contribution 
    ```
    {
      "label": "Appointments",
      "required": false,
      "id": "appointmentsWorkspaceLauncher",
      "questionOptions": {
        "rendering": "workspace-launcher",
        "buttonLabel": "Set the next appointment date",
        "workspaceName": "appointments-form-workspace"
      },
      "hide": {
        "hideWhenExpression": "true"
      }
    },
    ```

2.  Tooltip text partially hidden
This happens only hen the label has few characters ie short.

    **Solution**:
    - give a position to the tooltip, ie allow the tooltip to be displayed on the right in case the label is short
    - add styling to the tooltip to span in only the visible space.

### CODE REVIEW

File: sites/mosul/configs/openmrs/initializer_config/ampathforms/F03-mhGAP_Baseline.json [hunk 1]
```diff
@@ -727,7 +729,7 @@
             "questions": [
               {
                 "id": "cgiS",
-                "label": "CGI-S (rating of severity CGI-S how mentally ill is the patient at this time)",
+                "label": "CGI-S (rating of severity CGI-S how mentally ill is the patient at this time?) How",
```
[the remark this answers is not in the document]
Comment (jnsereko) [reply sites/mosul/configs/openmrs/initializer_config/ampathforms/F03-mhGAP_Baseline.json:732]: Screenshot 2024-09-05 at 12 35 41

I saw it in the ticket comments.

File: sites/mosul/configs/openmrs/initializer_config/ampathforms/F01-MHPSS_Baseline.json [hunk 2]
```diff
@@ -2001,19 +2001,19 @@
                     "concept": "5aa4edf5-b218-4470-b5da-ad1d88007954"
                   },
                   {
-                    "label": "Minimal symptoms",
+                    "label": "Minimal symptoms (5-9)",
```
[the remark this answers is not in the document]
Comment (pirupius) [reply sites/mosul/configs/openmrs/initializer_config/ampathforms/F01-MHPSS_Baseline.json:2004]: Yes

File: sites/mosul/configs/openmrs/initializer_config/ampathforms/F03-mhGAP_Baseline.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (pirupius) [reply sites/mosul/configs/openmrs/initializer_config/ampathforms/F03-mhGAP_Baseline.json:732]: Please check the meta file

File: sites/mosul/configs/openmrs/initializer_config/ampathforms/F01-MHPSS_Baseline.json [hunk 3]
```diff
@@ -127,7 +127,7 @@
               "type": "obs",
               "required": false,
               "questionOptions": {
-                "rendering": "number",
+                "rendering": "text",
```
[the remark this answers is not in the document]
Comment (jnsereko) [reply sites/mosul/configs/openmrs/initializer_config/ampathforms/F01-MHPSS_Baseline.json:130]: the skip logic only works when rendering is either a `text` or `text-area` so if its troublesome changing the concept datatypes, i can tag it as community and follow up.

File: sites/mosul/configs/openmrs/initializer_config/ampathforms/F02-MHPSS_Follow-up.json [hunk 4]
```diff
@@ -924,19 +925,19 @@
                       "concept": "5aa4edf5-b218-4470-b5da-ad1d88007954"
                     },
                     {
-                      "label": "Minimal symptoms",
+                      "label": "Minimal symptoms (5-9)",
```
[the remark this answers is not in the document]
Comment (jnsereko) [reply sites/mosul/configs/openmrs/initializer_config/ampathforms/F02-MHPSS_Follow-up.json:928]: not really,  I was confused with how to add ranges like i stated in https://github.com/MSF-OCG/LIME-EMR/pull/40#discussion_r1745112119

File: sites/mosul/configs/openmrs/initializer_config/ampathforms/F03-mhGAP_Baseline.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (pirupius) [reply sites/mosul/configs/openmrs/initializer_config/ampathforms/F03-mhGAP_Baseline.json:732]: The file has the correct version, for this case the missing question mark

Screenshot 2024-09-05 at 12 46 34

[the remark this answers is not in the document]
Comment (jnsereko) [reply sites/mosul/configs/openmrs/initializer_config/ampathforms/F03-mhGAP_Baseline.json:732]: its not there, why was it included here 🤔

File: sites/mosul/configs/openmrs/initializer_config/ampathforms/F01-MHPSS_Baseline.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (pirupius) [reply sites/mosul/configs/openmrs/initializer_config/ampathforms/F01-MHPSS_Baseline.json:130]: A fix for this has already merged to the `form-engine-lib`. Will make a PR to bump the lib in the `form-engine-app` later today to resolve this

**Review by pirupius — APPROVED** [review]

**Pull request merged**